emission/integrationTests/Dockerfile

```diff
@@ -0,0 +1,12 @@
+# python 3
+FROM ubuntu:latest
+
+RUN apt-get update
+RUN apt-get install -y curl
+
+# CHANGEME: Create the files that correspond to your configuration in the conf directory
+
+RUN echo "About to copy e-mission server code"
+COPY start_integration_tests.sh/ /start_integration_tests.sh
```

Comment:
and replace this with `setup/setup_tests.sh` instead

Reply:
Done